presto-accumulo/pom.xml

```diff
@@ -232,6 +232,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```

Comment:
This is exactly the version that used to be here -- https://github.com/airlift/airbase/pull/167/files#diff-f9e33823adeacc2b9a4de7229704d3f0L1134